troubleshoot/proxy/upstreams.go

```diff
@@ -0,0 +1,120 @@
+package troubleshoot
```

Comment:
Should these upstreams files be in troubleshoot/upstreams? I figured the structure would be something like 

```
troubleshoot
  proxy
    *files related to the troubleshoot proxy command*
  upstreams
    *files related to the troubleshoot upstreams command* 
```

Reply:
We use the same receiver in those files, separating them needs a refactor, will create a separate PR for that.